src/core/lib/surface/completion_queue.cc

```diff
@@ -847,10 +851,13 @@ static void cq_end_op_for_callback(
     cq_finish_shutdown_callback(cq);
   }
 
-  GRPC_ERROR_UNREF(error);
-
   auto* functor = static_cast<grpc_experimental_completion_queue_functor*>(tag);
-  grpc_core::ApplicationCallbackExecCtx::Enqueue(functor, is_success);
+  GRPC_CLOSURE_SCHED(
+      GRPC_CLOSURE_CREATE(functor_callback, functor,
+                          grpc_core::Executor::Scheduler(grpc_core::ExecutorJobType::SHORT)),
```

Comment:
Is SHORT or LONG a better choice here? I don't think it makes a difference in practice but I wonder if we're using them to mean something at the current time.

Reply:
It seems like the grpc_core::ExectuorJobType:: is used for maintaining stats about the type of call, doesn't seem to be used for any other purpose afaict. Maybe I should account for the worst case and use LONG?

Reply:
Moved to LONG now.